sdk/storage/azblob/zc_storage_error.go

```diff
@@ -70,22 +73,32 @@ func handleError(err error) error {
 }
 
 // defunkifyStorageError is a function that takes the "funky" ResponseError and reduces it to a storageError.
-func defunkifyStorageError(responseError ResponseError) error {
-	if err, ok := responseError.Unwrap().(*StorageError); ok {
-		// errors.Unwrap(responseError.Unwrap())
+func defunkifyStorageError(responseError *azcore.ResponseError) error {
```

Comment:
Not critical for this change, but this functional should be renamed.

Reply:
Fixed